.github/coordinator/test/lifecycle.test.mjs

```
@@ -116,17 +116,33 @@ const ENV = {
   DISPATCH_TARGET_SHA: SHA,
 };
 
-function makeRun(store, wa) {
+function makeRun(store, wa, extraIo = {}) {
   const out = [];
   return main([], ENV, {
     configPath: tempConfig(),
+          adapterModules: { "codex-cli": waCompat }, // SHU-63 pivot: builder lane routes to codex-cli
```

Biome (2.5.8)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)